dask/dataframe/methods.py

```diff
@@ -333,7 +330,7 @@ def pivot_count(df, index, columns, values):
 concat_dispatch = Dispatch("concat")
 
 
-def concat(dfs, axis=0, join="outer", uniform=False, filter_warning=True):
+def concat(dfs, axis=0, join="outer", uniform=False, filter_warning=True, **concat_kwargs):
```

Comment:
Lets just use `**kwargs` here and below.  I think that we don't need to specify the `concat_` bit in the name.

Reply:
Done